mpvc: 1.3 -> 1.4-unstable-2024-07-09

AndersonTorres opened the pull request:
## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

closes https://github.com/NixOS/nixpkgs/pull/286347

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

AndersonTorres committed:
mpvc: migrate to by-name

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-ready-for-review/3032/3434

AndersonTorres commented:
Hell, one month and no one cares!

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-already-reviewed/2617/1676

SuperSandro2000 commented on pkgs/by-name/mp/mpvc/package.nix:
```suggestion
    rm -r $out/share/
```

AndersonTorres commented on pkgs/by-name/mp/mpvc/package.nix:
OK.

SuperSandro2000 commented:
please rebase on master

AndersonTorres committed:
mpvc: rework

- align input set
- finalAttrs
- strictDeps
- remove nested with

AndersonTorres committed:
mpvc: 1.3 -> 1.4-unstable-2024-07-09

- Using a new repo because it has more updates;
- Using master tip instead of tagged release because, well, 1.5 does not exist
and the latest versions are `1.5-<some audio-related word>`.
- Adding myself as maintainer

AndersonTorres committed:
mpvc: remove neeasade as maintainer

No activity since 2023-07-20.

AndersonTorres commented:
Done.

AndersonTorres commented:
@SuperSandro2000
